server/graphql/v2/object/Host.js

```diff
@@ -427,6 +442,113 @@ export const Host = new GraphQLObjectType({
           };
         },
       },
+      contributionStats: {
+        type: new GraphQLNonNull(ContributionStats),
+        args: {
+          accounts: {
+            type: new GraphQLList(new GraphQLNonNull(AccountReferenceInput)),
+            description: 'A collection of accounts for which the contribution stats should be returned.',
+          },
+          from: {
```

Comment:
:ice_cream: We don't have a clear pattern for that in GQLV2, but I would personally opt for `dateFrom`/`dateTo` as we do in `TransactionsCollectionQuery`, since `from`/`to` could just as well refer to the accounts used for the contribution. @znarf any opinion on this?

Reply:
Yeah definitely `dateFrom` and `dateTo` like the transactions query. Never invent new names if they exist elsewhere.

After years in API design, my personal favorite is actually `since`/`before` but yeah, `dateFrom` and `dateTo` here.